### PR TITLE
Sto ssl doc notes doc 3580

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

- Enhanced Import Notes and new Troubleshooting tips section: [Adding Custom Certificates and Artifacts to STO Pipelines](https://64de5c81f3a3b4367fcb81be--harness-developer.netlify.app/docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines)
- Added note -- don't rename images in your private registry -- and a few other tweaks to [workflow description](https://64de5c81f3a3b4367fcb81be--harness-developer.netlify.app/docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/download-images-from-private-registry#workflow-description)

### CODE REVIEW

**Review by aklaft — DISMISSED** [review]
looks good. I like that you added all the notes to the top of the page

File: docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines.md [hunk 1]
```diff
@@ -286,3 +291,18 @@ pipeline:
...
+
+## Troubleshooting tips
+
+- To troubleshoot certificate issues, run your pipeline in Debug mode and check for log messages such as **`unable to get local issuer certificate`**.
+
+- In some cases, a scan might report that a certificate is invalid when in fact the root cause is not related to SSL. For example, the certificates might have an invalid domain defined. To determine if the root cause is SSL-related, you might try running a scan with SSL verification disabled temporarily. 
+
+  You'll need to disable verification in both the Harness pipeline and the external scanner. Note that not all scan tools support this option. 
+
+  -  For information about disabling SSL verification in the scanner, go to the external scanner documentation. If the includes a CLI option for this, you can use `tool_args` in your step to run a scan with this option turned off. For example, you can run a [Black Duck Hub](/docs/security-testing-orchestration/sto-techref-category/black-duck-hub-scanner-reference#settings) scan with this setting: `tool_args : --blackduck.trust.cert=TRUE`
```
Comment (brian-f-harness) [inline docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines.md:304]: The second sentence may be missing a word: If the includes

Comment (brian-f-harness) [inline docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines.md:304]:
```suggestion
  -  For information about disabling SSL verification in the scanner, go to the external scanner documentation. If the includes a CLI option for this, you can use `tool_args` in your step to run a scan with this option turned off. For example, you can run a [Black Duck Hub](/docs/security-testing-orchestration/sto-techref-category/black-duck-hub-scanner-reference#settings) scan with this setting: `tool_args : --blackduck.trust.cert=TRUE`.
```

File: docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines.md [hunk 2]
```diff
@@ -286,3 +291,18 @@ pipeline:
...
+ 
+  - If you're using a scanner-specific step with a scanner template, such as Aqua Trivy or Mend, uncheck **Enforce SSL** in the configuration palette. 
+
+  - If you're using a Security step without a scanner template, add this setting to the step: `bypass_ssl_check : true`
```
Comment (brian-f-harness) [inline docs/security-testing-orchestration/use-sto/set-up-sto-pipelines/add-artifacts-to-pipelines.md:308]:
```suggestion
  - If you're using a Security step without a scanner template, add this setting to the step: `bypass_ssl_check : true`.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Added a comment about a possible missing word.

**Pull request merged**